internal/cmd/flink/command_compute_pool_update.go

```diff
@@ -0,0 +1,96 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	flinkv2 "github.com/confluentinc/ccloud-sdk-go-v2/flink/v2"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *command) newComputePoolUpdateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "update [id]",
+		Short:             "Update a Flink compute pool.",
+		Args:              cobra.MaximumNArgs(1),
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validComputePoolArgs),
+		RunE:              c.computePoolUpdate,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Update Flink compute pool "my-compute-pool" to 2 CFUs.`,
+				Code: "confluent flink compute-pool update my-compute-pool --cfu 2",
+			},
+		),
+	}
+
+	cmd.Flags().Int32("cfu", 0, "Number of Confluent Flink Units (CFU).")
```

Comment:
Nit part 1: we should mark this required since it's the only modifiable parameter here, and it's essentially a no-op otherwise. Commands like `environment update` or `iam user update` do this too.

Reply:
When we add more flags in the future, it'll feel weird to make this non-required... but agree with preventing the no-op.